src/neighborhood.jl

```diff
@@ -36,6 +38,18 @@ struct Neighborhood
         allowed_mutations = filter(recipe -> !(recipe in frozen_shifts), get_nbhd(shifts))
         new(allowed_mutations, shifts)
     end
+
+    function Neighborhood(shifts::Shifts, frozen_days::Vector{Int}, sample_size::Real)
+        allowed_mutations = filter(recipe -> !(recipe.day in frozen_days), get_nbhd(shifts))
+        size = max(floor(Int, (length(allowed_mutations) * sample_size)), 1)
+        new(sample(allowed_mutations, size, replace=false), shifts)
+    end
+
+    function Neighborhood(shifts::Shifts, frozen_shifts::Vector{Tuple{Int, Int}}, sample_size::Real)
```

Comment:
Czy to musi być Real, nie da się bardziej zawęzić, w ogóle można by dawać jakieś sprawdzanie czy jest to float z range (0,1]

Reply:
Ponawiam pytanie, czy nie zmienić to na Float64 na przyklad?

Reply:
@Thematiq